scripts/grep-forge.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```

Comment:
this script is an old artefact. please, remove it.

Reply:
Done.